### ISSUE TITLE
Where are you? CocosBuilder's founder

### ISSUE THREAD

**Issue by wolfand11** [cocos2d/CocosBuilder#419]
It's three months since CocosBuilder was updated. Is there any message about CocosBuilder's future?

**Comment by danielhaaser** [comment]
Viktor took a job at Apportable.  https://www.apportable.com/team

As far as I know, there hasn't been any announcement about the future of CocosBuilder.

**Comment by apostoiis** [comment]
Yes it would be a newer version of CocosBuilder probably on October..with quite a lot of changes there..Stay tuned :dancer:

**Comment by Ricardo1980** [comment]
In that case, is this project still alive Apostolis?

**Comment by apostoiis** [comment]
I am not sure if this is the actually working project, I just know that it will be a new version soon, with a different name ( SpriteBuilder probably ) and a lot new features in.

**Comment by Ricardo1980** [comment]
Thanks for reply.
Do you know who is creating that version? SpriteBuilder? Is that for cocos2d or for SpriteKit? Is zynga involved?
Any info is appreciated. Thanks.

**Comment by nickveri** [comment]
You can find out more about SpriteBuilder in this thread http://www.cocos2d-iphone.org/forums/topic/spritebuilder-sneak-peek/

It's being developed by Viktor Lidholt (the author of CocosBuilder) who now works for Apportable. It looks great, however, it's a pity that they dropped the JavaScript related features (it's understandable though).

I think it's safe to assume that CocosBuilder is dead. It would be great to know if the cocos2d-x team is planning to support SpriteBuilder.

**Comment by Ricardo1980** [comment]
Wonderful! I didn't know anything about that. Thanks.

**Comment by collinjackson** [comment]
You can follow the development of SpriteBuilder here: https://github.com/apportable/SpriteBuilder

**Comment by shayneoneill** [comment]
Maybe someone should fork Cocosbuilder into a community branch, because whilst Spritebuilder looks great, it doesnt support cocos2d-x (And the cocosStudio mysteriously doesnt support the mac!)

**Comment by KevinLion** [comment]
There is an important task, so spriteBuilder support cocos2d-X, who is the hero to complete it?

**Comment by edudemy** [comment]
:+1:

**Comment by nguyenduong** [comment]
Hi bros. SpriteBuilder lib for cocos2d-x is ready. Please take some minutes to visit my page:
http://jacos2d-x.org.
Thanks:)

**Comment by beckheng** [comment]
Sound great!!!

**Comment by edudemy** [comment]
Great! Does this mean it supports javascript now? Cocos2d-x JS Bindings or Cocos2d-HTML5?

**Comment by nguyenduong** [comment]
Sorry, It doesn't support JS now.

**Comment by edudemy** [comment]
Okay. How Is this [project](https://github.com/nguyenduong/spritebuilder_cocos2dx) different from the other CCBReader project?
https://github.com/growingdever/cocos2dx-ccbreader

**Comment by nguyenduong** [comment]
It support SpriteBuilder version 1.0.6 fully: Physic, BoxLayout,...
